libraries/default.rb

```diff
@@ -471,6 +471,18 @@ def install_live_streaming_service_config(current_deploy_root,live_stream_name)
       end
     end
 
+    def install_aws_s3_distribution_service_config(current_deploy_root, region, s3_distribution_bucket_name)
+      template %Q|#{current_deploy_root}/etc/services/edu.harvard.dce.distribution.aws.s3.AwsS3DistributionServiceImpl.properties| do
+        source 'edu.harvard.dce.distribution.aws.s3.AwsS3DistributionServiceImpl.properties.erb'
+        owner 'matterhorn'
+        group 'matterhorn'
+        variables({
+          region: region,
+          s3_distribution_bucket_name: s3_distribution_bucket_name 
+        })
+      end
+    end
+
```

Comment:
Will all clusters (dev or not) require an s3 distribution bucket name from now on?  

Put another way - if this service isn't configured, will a cluster "just work?" If not, then I'll need to think about backwards compatibility and how to automate bucket creation during creation.